src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/QuestionManager.java

```diff
@@ -501,15 +502,15 @@ public Map<TimeInterval, Map<Role, Long>> getAnsweredQuestionRolesOverPrevious(
    * @return a Map of LocalDates to attempt counts
    */
   public Map<LocalDate, Long> getUsersQuestionAttemptCountsByDate(final RegisteredUserDTO user,
-                                                                  final Date fromDate, final Date toDate,
+                                                                  final Instant fromDate, final Instant toDate,
                                                                   final Boolean perDay) throws SegueDatabaseException {
-    Map<Date, Long> questionAttemptCountPerDateByUser = this.questionAttemptPersistenceManager
+    Map<Instant, Long> questionAttemptCountPerDateByUser = this.questionAttemptPersistenceManager
         .getQuestionAttemptCountForUserByDateRange(fromDate, toDate, user.getId(), perDay);
 
     // Convert the normal java dates into useful joda dates and create a new map.
```

Comment:
this comment is now incorrect

Reply:
Ah, overlooked the comment. I'll see about sorting that...